manuscript/1-SoftReq.txt

```diff
@@ -790,7 +787,7 @@ models), καταστατικά μοντέλα (state models), μοντέλα μ
 επίτευξη συμφωνίας μεταξύ των πελατών και των αναδόχων ή προμηθευτών
 (σε έργα με γνώμονα την αγορά, αυτοί οι ρόλοι μπορούν να αναπαρασταθούν
 από τα τμήματα μάρκετινγκ και ανάπτυξης) σε ό,τι το προϊόν λογισμικού
-προσδοκάται ή όχι να κάνει.
+προσδοκάτε ή όχι να κάνει.
```

Comment:
Please keep passive voice.  There are no hints of active voice in the preceding paragraph.